content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,133 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
 
 - [Accelerate](/accelerate/what-is-accelerate): Global database cache with scalable connection pooling
 - [Pulse](/pulse/what-is-pulse): A managed Change Data Capture (CDC) service that enables real-time database subscriptions
 
-[Go to Platform Console](https://console.prisma.io)
+Open the [Platform Console](https://console.prisma.io).
 
-</TopBlock>
+## Concepts
+
+The Platform Console workflows are based on four main concepts:
+
+- [**Account**](#account): In order to use PDP products, you need to have a PDP account. An _organization_ will typically create one account to manage all their workspaces, projects and environments.
+- [**Workspaces**](#workspace): An account can have multiple workspaces. A workspace typically represents a _team_ within an _organization_. **Billing happens on a per-workspace basis**, i.e. an _organization_ receives an invoice at the end of the month that captures all cost associated with the projects in a given workspace.
+- [**Project**](#project): A workspace can have multiple projects. A project typically represents the _application_ of a _product_ or _service_ a team is working on.
+- [**Environments**](#environment): A project can have multiple environments. An environment typically maps to a _development stage_, like `dev`, `staging` or `prod`. **API keys are provisioned on the environment-level**.
+
+Here is a visual illustration of how these concepts relate to each other:
+
+![](/img/platform/pdp-concepts.png)
+
+### Account
+
+Your account is the basis for all interactions with PDP products. You can use it to manage your workspaces (and their projects).
+
+If you need to delete your account, go [here](/platform/support#deleting-your-pdp-account).
+
+### Workspace
+
+In your PDP account, you can create several workspaces. A workspace is an isolated space to host projects in which you use PDP products.
+
+In each workspace, you can:
+
+- view and manage all projects (and their environments) in that workspace.
+- manage billing, i.e. select a [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site), configure payment methods, or view the invoice history.
+- view the usage of your enabled PDP products across all projects in that workspace.
+- manage team members and change their roles / permissions in the workspace.
+
+### Project
+
+In each workspace, you can create several projects. A project typically represents an application (a product or service) your organization is offering. You typically have one [Prisma schema](/orm/prisma-schema/) per project.
+
+In each project, you can:
+
+- view and manage all environments in that project.
+
+The number of project you can create in a workspace depends on the [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) configured in that workspace:
+
+Here's an overview of the number of projects per workspace on each plan:
```

Comment:
I see the conversation above :) I'd actually remove this, and just let people look on the pricing page to minimize the places where we have a source of truth. If we ever decide to change this, it'll be a pain to find all places we mention explicit numbers.

Reply:
I thought it's useful because IMO it helps to strengthen the understanding of workspaces, projects and environments and how they relate to each other (and also because Nurul brought it up as something people are confused about and that we may want to document).

> If we ever decide to change this, it'll be a pain to find all places we mention explicit numbers.

I'd take the tradeoff of better/clearer explanations and needing to go through the docs to update pricing information in favor over not showing this info.

Do you feel strongly about removing this information? If so, happy to do it of course!

Reply:
If you guys think it's valuable, let's leave it. I can see your reasoning to strengthen understanding. It's likely not something we'll change a lot.